src/gtclang/Driver/Options.inc

```diff
@@ -46,7 +46,8 @@ OPT(std::string, ConfigFile, "", "config", "",
 OPT(std::string, Backend, "gridtools", "backend", "", 
     "Set backend code-generator. Possible values for <backend> are:"
     "\n - gridtools = gridtools C++ code"
-    "\n - c++       = naive C++ code\n", "<backend>", true, false)
+    "\n - c++-naive   = naive C++ code\n"
```

Comment:
should we maybe call it c++simplified?

Reply:
not sure, maybe also `c++-sequential`, because the main point of this backend is that is not parallelized. But can we discuss and change it later? Cause I will have to rename all the work in dawn, namespaces, etc.

Reply:
sure, this discussion is not urgent